### ISSUE TITLE
Spell check should ignore mid-word tags

### ISSUE THREAD

**Issue by nianiam** [notaapp/nota#165]
If I were to write \**P**rojects (interpreted as **P**rojects), the spell check sees this as two separate words and underlines 'rojects' because that isn't a word. Obviously this makes sense if you can see the tags, however, if you want to hide the markdown you're suddenly going to get a load of 'mis-spelled' words. This could be solved by having the spell check ignore tags within a single word (ie between two bits of whitespace).

This would break the opposite case where someone would want to write separate words separated by some tags (e.g. really**really**bold), but this seems like the rarer edge case!

**Comment by astoilkov** [comment]
Yes. I agree. That makes sense. I've been thinking about such an enhancement in the past. Thanks!
